dwave/cloud/regions.py

```diff
@@ -22,18 +22,19 @@
 
 from dwave.cloud import api
 from dwave.cloud.api.constants import (
-    DEFAULT_REGION, DEFAULT_SOLVER_API_ENDPOINT, DEFAULT_LEAP_API_ENDPOINT)
+    DEFAULT_REGION, DEFAULT_SOLVER_API_ENDPOINT, DEFAULT_LEAP_API_ENDPOINT,
+    DEFAULT_METADATA_API_ENDPOINT)
 from dwave.cloud.config.models import ClientConfig, validate_config_v1
 from dwave.cloud.utils import cached
 
 __all__ = ['get_regions']
 
 logger = logging.getLogger(__name__)
 
-_REGIONS_CACHE_MAXAGE = 86400   # 1 day
+_REGIONS_CACHE_MAXAGE = 7 * 86400   # 7 days
```

Comment:
Maybe one of these locations should import the same value from the other - single source of truth etc etc.

Reply:
They are independent. One is for general `get_regions` cache, and the other is specifically for calls from `Client`. The client can override the maxage, but for now I just kept them the same.